src/signup.html

```diff
@@ -33,8 +33,8 @@
       </div>
       <!-- <textarea></textarea> -->
       <div class="form-group">
-        <label>Write About Yourself</label>
-        <textarea name="biography"></textarea>
+        <label for="bio">Write About Yourself</label>
+        <textarea name="biography" id="bio"></textarea>
```

Comment:
Add a namespace to the ID here as well :)

Reply:
Ahh correct ! I've just updated it.